src/sentry/api/serializers/models/project.py

```diff
@@ -149,8 +150,28 @@ def get_features_for_projects(
     return features_by_project
 
 
+class ProjectSerializerResponse(TypedDict):
+    id: str
+    slug: str
+    name: str  # TODO: add deprecation about this field (not used in app)
+    isPublic: bool
+    isBookmarked: bool
+    color: str
+    dateCreated: str
+    firstEvent: Any  # TODO
+    firstTransactionevent: bool
+    hasSessions: bool
+    features: Any  # TODO
```

Comment:
@JoshFerge isn't this an array of strings? Seems easy to type.

Reply:
good point. updated.